src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -224,12 +241,52 @@ public int getVtgatePort() {
         return getConfig().getInteger(VTGATE_PORT);
     }
 
+    private void checkVtgateAuthFile() {
```

Comment:
Maybe name it something like `loadVtgateAuthFile()` since you are actually loading the credential from file into memory.

Reply:
I can name the method: 'maybeLoadVtgateAuthFile' since it's conditional loading.